mitodl/odl-video-service
# Enabled video edit button on collection detail page

#### What are the relevant tickets?
Closes #188

#### What's this PR do?
Adds the share button back to the video card on collection detail

#### How should this be manually tested?
Click the edit button on both the collection detail and video detail pages, change some stuff, click "save". Also, add/remove your admin permissions for the given collection and test that the edit button appears or hides based on that permission

#### Where should the reviewer start?
`EditVideoFormDialog.js`

#### Any background context you want to provide?
The 'share' button will be enabled in a different PR.

#### Screenshots (if appropriate)
![ss 2017-09-11 at 14 06 45](https://user-images.githubusercontent.com/14932219/30289540-7fb4e8fe-96fa-11e7-84c5-d32e993e7bf7.png)
